samples_project/ProjectSettings/TagManager.asset

```diff
@@ -3,7 +3,9 @@
 --- !u!78 &1
 TagManager:
   serializedVersion: 2
-  tags: []
+  tags:
+  - FeatureItem
```

Comment:
What do these tags do? Is there a way to do the same thing without adding tags?

Reply:
It allows searching for specific items with that tag. In Unreal there is a "Get All Items of Class" that I was able to use to do that something but you can't do that with unity. This was the easiest solution from my research

Reply:
Hmm, what about `FindObjectsOfType()`? 
https://docs.unity3d.com/ScriptReference/Object.FindObjectsOfType.html